wiki/en/Include-Shared-Commands.md

```diff
@@ -2,10 +2,13 @@
 [comment]: # (This is an include file for use in multiple documents)
 
 - `-h` or `--help`     Display help text         
-- `-i` or `--inifile`   Set location of initialization file (overrides default) 
+- `-i` or `--inifile`   Set location of initialization file (overrides default)
```

Comment:
Spacing?

Reply:
And line 6?

Reply:
Spacing is wrong in the whole file, I think?

Reply:
I guess the spaces get collapsed, anyway.  And `.md` and tables don't always get on...

Reply:
MD tables should work?